### DISCUSSION TITLE
nlnet - two applicable funding opportunities, June 1st deadline

### DISCUSSION THREAD

**Discussion by matkoniecz** [organicmaps/organicmaps#7907] [Ideas]
- NGI0 Commons Fund: Reclaim the public nature of the internet
- NGI MobiFree: More ethical and human mobile software

It seems that OM (or more specifically, specific OM improvements projects) may fit well.

Note: obviously if anyone wants to send proposal they should do it in consultation with OM authors.

nlnet previously appeared at least on https://github.com/orgs/organicmaps/discussions/4111

**Comment by vng** [comment]
We already have NLNet grant, check out README file.

**Reply by matkoniecz** [reply]
Oh, nice it worked out. I admit that I missed this.

(not sure do you need to complete that grant to be able to propose other OM-adjacent grants - it may be necessary, and maybe completing this specific grant before 1st June is unfeasible)
